index.html

```diff
@@ -80,7 +80,156 @@ <h1 class="mn__title">
+              <a class="bt" href="#">JavaScript</a>
+              <a class="bt" href="#">HTML</a>
+            </div>
+            <a class="see__proj__btn btns" href="#"><span>See Project</span></a>
+          </div>
+        </div>
+        <div class="proj__container" id="proj__2">
+          <img
+            class="proj__img"
+            src="img/Work 2.webp"
+            alt="2nd project image"
+          />
+          <div class="proj__list">
+            <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+            <div class="proj__lang">
+              <a class="bt" href="#">Ruby on rails</a>
+              <a class="bt" href="#">CSS</a>
+              <a class="bt" href="#">JavaScript</a>
+              <a class="bt" href="#">HTML</a>
+            </div>
+            <a class="see__proj__btn btns" href="#"><span>See Project</span></a>
+          </div>
+        </div>
+        <div class="proj__container" id="proj__3">
+          <img
+            class="proj__img"
+            src="img/Work 3.webp"
+            alt="3rd project image"
+          />
+          <div class="proj__list">
+            <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+            <div class="proj__lang">
+              <a class="bt" href="#">Ruby on rails</a>
+              <a class="bt" href="#">CSS</a>
+              <a class="bt" href="#">JavaScript</a>
+              <a class="bt" href="#">HTML</a>
+            </div>
+            <a class="see__proj__btn btns" href="#"><span>See Project</span></a>
+          </div>
+        </div>
+        <div class="proj__container" id="proj__4">
+          <img class="proj__img" src="img/Work 4.jpg" alt="4th project image" />
+          <div class="proj__list">
+            <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+            <div class="proj__lang">
+              <a class="bt" href="#">Ruby on rails</a>
+              <a class="bt" href="#">CSS</a>
+              <a class="bt" href="#">JavaScript</a>
+              <a class="bt" href="#">HTML</a>
+            </div>
+            <a class="see__proj__btn btns" href="#"><span>See Project</span></a>
+          </div>
+        </div>
+        <div class="proj__container" id="proj__5">
+          <img
+            class="proj__img"
+            src="img/Work 5.webp"
+            alt="5th project image"
+          />
+          <div class="proj__list">
+            <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+            <div class="proj__lang">
+              <a class="bt" href="#">Ruby on rails</a>
+              <a class="bt" href="#">CSS</a>
+              <a class="bt" href="#">JavaScript</a>
+              <a class="bt" href="#">HTML</a>
+            </div>
+            <a class="see__proj__btn btns" href="#"><span>See Project</span></a>
+          </div>
+        </div>
+        <div class="proj__container" id="proj__6">
+          <img class="proj__img" src="img/Work 6.jpg" alt="6 project image" />
```

Comment:
:red_circle: [REQUIRED] Kindly make sure you spell the image name correctly.. As requested by the previous reviewer the src has an error, instead of calling img/Work_6.jpg you should use img/work_6.jpg. This line of code should be :arrow_down: 
`<img class="proj__img" src="./img/work 6.jpg" alt="6 project image" />`

Also when naming your files I think it is considered as a best practice to use underscore or dash instead of an empty(white) space :hugs:

Reply:
oh ok thanks, but the original image file name was Work 6.jpg that's why...
but now I have even corrected the naming just as per your recommendation Thanks again.